package-lock.json

```diff
@@ -6355,7 +6355,8 @@
     "detect-libc": {
       "version": "1.0.3",
       "resolved": "https://registry.npmjs.org/detect-libc/-/detect-libc-1.0.3.tgz",
-      "integrity": "sha1-+hN8S9aY7fVc1c0CrFWfkaTEups="
+      "integrity": "sha1-+hN8S9aY7fVc1c0CrFWfkaTEups=",
```

Comment:
Could you please ensure you're running node version `10.15.3` & then re-run the `npm install` - You should see that then these `optional true` changes aren't needed on the branch.